File: Java-Core/src/main/java/prompto/java/JavaLiteral.java
package prompto.java;

import prompto.utils.CodeWriter;

public abstract class JavaLiteral implements JavaExpression {
	
	String text;
	
	protected JavaLiteral(String text) {
		this.text = text;
	}
	
	@Override
	public void toDialect(CodeWriter writer) {
		writer.append(text);
	}
}

File: Java-CodeServer/src/main/resources/js/prompto/java/JavaLiteral.js
JavaLiteral = function(text) {
    this.text = text;
    return this;
};

JavaLiteral.prototype.toString = function() {
    return this.text;
};

JavaLiteral.prototype.toDialect = function(writer) {
    writer.append(this.text);
};


exports.JavaLiteral = JavaLiteral